test/AdoNetCore.AseClient.Tests/Unit/LoginPacketTests.cs

```diff
@@ -62,7 +61,25 @@ public void Matches_AdoNet4_Output()
             var adoNet4Packet = "44 45 53 2d 50 43 30 4b 46 55 53 48 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 0c 55 53 45 52 4e 41 4d 45 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 08 50 41 53 53 57 4f 52 44 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 08 39 33 36 30 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 04 03 01 06 0a 09 01 00 00 00 00 00 00 00 00 00 00 41 50 50 4e 41 4d 45 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 07 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 08 50 41 53 53 57 4f 52 44 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 0a 05 00 00 00 41 44 4f 2e 4e 45 54 00 00 00 07 0f 07 00 0d 00 0d 11 75 73 5f 65 6e 67 6c 69 73 68 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 0a 01 00 00 00 00 08 00 00 00 00 00 00 00 00 69 73 6f 5f 31 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 05 01 35 31 32 00 00 00 03 00 00 00 00 e2 20 00 01 0e 01 ef ff 69 b7 fd ff af 65 41 ff ff ff d6 02 0e 00 00 00 00 00 88 40 00 01 02 48 00 00 00";
             using (var ms = new MemoryStream())
             {
-                new LoginPacket("DES-PC0KFUSH", "USERNAME", "PASSWORD", "9360", "APPNAME", "", "us_english", "iso_1", "ADO.NET", 512, new CapabilityToken()).Write(ms, Encoding.ASCII);
+                new LoginPacket(
+                    "DES-PC0KFUSH",
```

Comment:
@senseibaka - this name shouldn't be in here.

Reply:
Woops